exercises/chapter2/test/Main.purs

```diff
@@ -19,11 +19,13 @@ main = do
         Assert.equal 233168 (answer 1000)
     {-  Move this block comment starting point to enable more tests
 Note to reader: Delete this line to expand comment block -}
+-- ANCHOR: diagonalTests
     suite "diagonal" do
       test "3 4 5" do
         Assert.equal 5.0 (diagonal 3.0 4.0)
       test "5 12 13" do
         Assert.equal 13.0 (diagonal 5.0 12.0)
+-- ANCHOR_END: diagonalTests
```

Comment:
Now that I'm looking at this with fresh eyes after not having touched the book for a while, I can image this section being really confusing to new readers if they glossed over the instructions to run the `resetSolutions.sh` and `removeAnchors.sh` scripts described [here](https://book.purescript.org/chapter2.html#solving-exercises). Wondering if this line:
```
Note to reader: Delete this line to expand comment block
```
should be changed to something like the following for every chapter:
```
Note to reader: This line should have been automatically deleted by resetSolutions.sh. See Chapter 2 for instructions.
```